requirements/requirements.txt

```diff
@@ -1,4 +1,3 @@
-fiddle
```

Comment:
I assume this is gone because it's pulled in by nemo_run now. I don't see how you're adding nemo_run as a dependency though. How does that happen now? I ask since we'll be making similar changes soon to bionemo.

Reply:
Yeah we will hold off on merging this PR until we have published the pypi package of nemo-run. When that's done it will replace fiddle in the requirements (nemo-run pulls in fiddle).

Reply:
Let's keep fiddle and add `nemo_run @ git+https://github.com/NVIDIA/NeMo-Run.git` separately for now. I will create a separate PR to move to PyPI later.